nodejs/listeners/actions/block_open_task_list_home.js

```diff
@@ -9,6 +10,30 @@ module.exports = (app) => {
     if (action.selected_options.length > 0) {
       const tasksToUpdate = action.selected_options.map((option) => option.value.split('-')[2]);
       Task.update({ status: 'CLOSED' }, { where: { id: tasksToUpdate } });
+      // Find all the tasks provided where we have a scheduled message ID
+      const tasks = await Task.findAll(
+        {
+          where: {
+            [Op.and]: [
+              { id: tasksToUpdate },
+              { scheduledMessageId: { [Op.not]: null } },
+            ],
+          },
+        },
+      );
+      tasks.map(async (task) => {
+        // If a reminder is scheduled, cancel it and remove the ID from the datastore
+        try {
+          await client.chat.deleteScheduledMessage({
+            channel: body.user.id,
+            scheduled_message_id: task.scheduledMessageId,
+          });
+          Task.update({ scheduledMessageId: null }, { where: { id: task.id } });
+        } catch (error) {
+          // eslint-disable-next-line no-console
+          console.error(error);
+        }
+      });
```

Comment:
Given this logic is repeated in `block_button-mark-as-done`, we could probably abstract it.